bin/merge

```diff
@@ -0,0 +1,54 @@
+#!/bin/bash
+
+# FIXME: Had to remove -e for the `read <(curl | json)` line to work
+set -uo pipefail # STRICT MODE
+IFS=$'\n\t'       # http://redsymbol.net/articles/unofficial-bash-strict-mode/
+
+# Depends on: curl, json
+
+# Takes two arguments – a GitHub repository name and a PR number
+# It ensures the PR is unmerged and in the ready state, and merges it into
+# master after updating the version number
+
+repo=${1:-}
+pull_request=${2:-}
+if [[ -z "$repo" || -z "$pull_request" ]]; then
+    echo "usage: $0 REPO PR"
+    exit 1
+fi
+
+pr_url="https://api.github.com/repos/${repo}/pulls/${pull_request}"
+
+directory=$(mktemp -d "/tmp/slamdata-merge.XXXXXXXX")
+
+read -d '' merged base_clone pr_clone submitter base_ref pr_ref \
+     < <(curl -f $pr_url | /usr/local/bin/json merged base.repo.clone_url head.repo.clone_url user.login base.ref head.ref)
+
+if [[ "$merged" == "false" ]]; then
+    tmp_branch=${submitter}-${pr_ref}
+
+    git clone $base_clone $directory
+    cd $directory
+    git checkout -b $tmp_branch $base_ref
+    git pull $pr_clone $pr_ref
+    git checkout $base_ref
+    git merge --no-ff --no-edit $tmp_branch
+    old_version=($(sed 's/.*"\(.*\)"/\1/' version.sbt | tr "." "\n"))
+    echo $old_version
+    if [[ -e "CHANGELOG/major.md" ]]; then
+        new_version="${old_version[0]+1}.0.0"
+        mv CHANGELOG/major.md CHANGELOG/${new_version}.md
+    elif [[ -e "CHANGELOG/minor.md" ]]; then
+        new_version="${old_version[0]}.${old_version[1]+1}.0"
+        mv CHANGELOG/minor.md CHANGELOG/${new_version}.md
+    else
```

Comment:
Better to be pedantic lest we get it wrong if someone misspells `minr` or mjaor`?